changelog.md

```diff
@@ -11,6 +11,10 @@ This is a maintenance release containing mostly bud fixes.
   * Make QuartzOptions Triggers and JobDetails public (#981)
   * Fix configuration system injection for dictionary/quartz.jobStore.misfireThreshold in DI (#983)
 
+* NEW FEATURE
```

Comment:
Can we move this to bug fix side as I think this rather a bug? "XMLSchedulingDataProcessor can throw IOException due to file locking" or something similar?

Reply:
It proves that it makes sence to discuss an issue before opening PR :)

I was not sure whether it is a feature to bug fix. Still I think it is not a bug in this code, but I agree that the change is better to specify under FIXES.

The changelog was undeted in f5e549a